docs/source/getting_started/1_pycharm.rst

```diff
@@ -27,71 +26,116 @@ Project Preparation
 
 #. Start the IDE and create a new project in the ``my-module`` directory.
 
-#. When opening the project, skip the step related to the environment creation: click **Cancel** on **Creating Virtual Environment**:
+#. When creating a project, select the system Python interpreter
 
-.. image:: ../_static/img/dev-env/01-cancel-creating-virtual-environment.png
+.. image:: ../_static/img/dev-env/01-creating-project.png
 
 Setting Up The Interpreter
 --------------------------
+#. Open the project settings with **File > Settings ...** or **Ctrl+Alt+S**. Set **Use compose V2** in **Build, Execution,Deployment > Docker > Tools**
+
+    .. image:: ../_static/img/dev-env/21-select-docker-compose-v2.png
 
 #. Open the project settings with **File > Settings ...** or **Ctrl+Alt+S**.
 
-#. Choose **Project: my-module > Python Interpreter** section.
+#. Select **Project: my-module > Python Interpreter** section.
 
-#. Add a new **Python Interpreter** by clicking the **Add Interpreter**, and choosing **On Docker...**:
+#. Add a new **Python Interpreter** by clicking the **Add Interpreter**, and selecting **On Docker Compose...**:
 
     .. image:: ../_static/img/dev-env/02-setup-python-interpreter.png
 
 #. In the modal window:
 
-  * **[Step 1 of 3]**: Choose Dockerfile: ``docker/Dockerfile.x86`` or ``docker/Dockerfile.l4t``, if you are on Jetson, and set up Context folder (should be set to the root of the project). In the **Optional** section you have to specify an **Image tag**:
+  * **[step 1 of 4]**: Click on the three dots under **Server** and set up access to the docker service
```

Comment:
On the image below: "1/3. Create a new ...". Are you sure there are 4 steps?

Reply:
made nested list